src/args.c

```diff
@@ -616,6 +616,15 @@ void parse_args(int argc, char *argv[])
 		// Enable stdout printing
 		cli_mode = true;
 		log_ctrl(false, true);
+
+		// Validate the output filename
+		if(strstr(argv[2], "..") || strchr(argv[2], '/') || strchr(argv[2], '\\'))
```

Comment:
Does this mean, the `/` character is not allowed?

Reply:
It depends. When we really only ever want to use this internally during CI it's fine. But I fear users will find this and try to do `pihole-FTL create-default-config /home/pi/subdir/toml.conf`